AnnotationsAndModifiers.md

```diff
@@ -0,0 +1,26 @@
+# Annotations and Modifiers
+
+## Annotations
+
+Method/variable annotations should go on the _same_ line as the rest of the definition. i.e.
+
+```swift
+@objc func doThing() { }
+```
+
+NOT
+
+```
+@objc
```

Comment:
Sometimes if you want to customize the generated objc name, putting the annotation on a separate line makes it more readable. Could add this as a separate case?

Reply:
Ah, good point. We do allow this to be on a separate line.